command/genesis/params.go

```diff
@@ -58,6 +61,9 @@ type genesisParams struct {
 	blockGasLimit uint64
 	isPos         bool
 
+	minNumValidators uint32
+	maxNumValidators uint32
```

Comment:
Why do you accept only `uint32`s for this? There is no reason to limit the user, nobody in their right mind would choose to write `1073741824` over `0x40000000`.

Why not accept a `uint32` or a hex value?

Reply:
Which code?

I don't see you accepting string values and parsing them:
https://github.com/0xPolygon/polygon-edge/blob/731e69945d08d40242af8d7e5f032736c093c0d5/command/genesis/genesis.go#L131-L142

Reply:
Cobra handles it internally. I have tried using hex as a value for any of these flags and it parses it correctly.